prdoc/pr_5857.prdoc

```diff
@@ -0,0 +1,19 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Added benchmarks for BEEFY fork voting
```

Comment:
```suggestion
title: "Beefy equivocation: check all the MMR roots"
```

Reply:
Thanks ! Missed that. Started from an old prdoc.